src/features/BurgerNavigation/index.tsx

```diff
@@ -0,0 +1,2 @@
+import BurgerNavigation from "./BurgerNavigation";
+export default BurgerNavigation;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.